tests/shared/drive-constraints.spec.ts

```diff
@@ -700,11 +700,6 @@ describe('Shared: DriveConstraints', function () {
 				});
 
 				it('should return false if the drive is not large enough and is a source drive', function () {
-					console.log('YAYYY', {
-						...image,
-						path: path.join(this.mountpoint, 'rpi.img'),
-						size: 5000000000,
-					});
```

Comment:
Please refactor the PR to remove this commit and avoid adding this console.log in the first place.

Reply:
It was merged into master, we would have to rebase that which I'm really skeptical about